### PR TITLE
Add test on JMAP message creation

### PR DESCRIPTION
**Pull request by chibenwa**
Demonstrate it works with arbitrary message ID.

### CODE REVIEW

**Comment by aduprat** [comment]
:+1:

File: server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java [hunk 1]
```diff
@@ -712,6 +712,59 @@ public void setMessageShouldReturnCreatedMessageWhenSendingMessage() {
...
+            "        \"subject\": \"Thank you for joining example.com!\"," +
+            "        \"textBody\": \"Hello someone, and thank you for joining example.com!\"," +
+            "        \"mailboxIds\": [\"" + getOutboxId() + "\"]" +
+            "      }}" +
+            "    }," +
+            "    \"#0\"" +
+            "  ]" +
+            "]";
+
+        given()
```
Comment (mbaechler) [inline server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java]: you don't need all these assertions, you should only check that the server is ok with your specific messageId

**Comment by chibenwa** [comment]
Ok

File: server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java [hunk 2]
```diff
@@ -712,6 +712,41 @@ public void setMessageShouldReturnCreatedMessageWhenSendingMessage() {
...
+            .accept(ContentType.JSON)
```
Comment (mbaechler) [inline server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java]: I guess you no longer need that on master, do you ?

Comment (chibenwa) [inline server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java]: Yes

**Comment by mbaechler** [comment]
:+1:

**Comment by rouazana** [comment]
:+1:

**Comment by chibenwa** [comment]
Just commited

**Pull request closed without merging**